lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -243,17 +181,6 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
                  v
                }
 
-  option :tags,
-         long: "--tags TAGS",
-         description: "Comma separated list of tags to apply to the node",
-         proc: ->(tags) { tags.split(/[\s,]+/) },
-         default: []
-
-  option :bootstrap_template,
-         short: "-t TEMPLATE",
-         long: "--bootstrap-template TEMPLATE",
-         description: "Bootstrap Chef using a built-in or custom template. Set to the full path of an erb template or use one of the built-in templates."
-
   option :template_file,
```

Comment:
This could be removed too because we are using `:bootstrap_template` now.

Reply:
removed `template_file` option.